src/event.rs

```diff
@@ -835,11 +904,74 @@ pub enum ModifierKeyCode {
+    ///
+    /// # Platform-specific Notes
+    ///
+    /// On macOS, the control, alt, and super keys is displayed as "Control", "Option", and
+    /// "Command" respectively. See
+    /// <https://support.apple.com/guide/applestyleguide/welcome/1.0/web>.
+    ///
+    /// On Windows, the super key is displayed as "Windows" and the control key is displayed as
+    /// "Ctrl". See
+    /// <https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/term-collections/keys-keyboard-shortcuts>.
+    ///
+    /// On other platforms, the super key is referred to as "Super".
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        #[cfg(target_os = "macos")]
+        match self {
+            ModifierKeyCode::LeftShift => write!(f, "Left Shift"),
+            ModifierKeyCode::LeftHyper => write!(f, "Left Hyper"),
+            ModifierKeyCode::LeftMeta => write!(f, "Left Meta"),
+            ModifierKeyCode::RightShift => write!(f, "Right Shift"),
+            ModifierKeyCode::RightHyper => write!(f, "Right Hyper"),
+            ModifierKeyCode::RightMeta => write!(f, "Right Meta"),
+            ModifierKeyCode::IsoLevel3Shift => write!(f, "Iso Level 3 Shift"),
+            ModifierKeyCode::IsoLevel5Shift => write!(f, "Iso Level 5 Shift"),
+
+            #[cfg(target_os = "macos")]
+            ModifierKeyCode::LeftControl => write!(f, "Left Control"),
+            #[cfg(not(target_os = "macos"))]
+            ModifierKeyCode::LeftControl => write!(f, "Left Ctrl"),
+
+            #[cfg(target_os = "macos")]
+            ModifierKeyCode::LeftAlt => write!(f, "Left Option"),
+            #[cfg(not(target_os = "macos"))]
+            ModifierKeyCode::LeftAlt => write!(f, "Left Alt"),
+
+            #[cfg(target_os = "macos")]
+            ModifierKeyCode::LeftSuper => write!(f, "Left Command"),
+            #[cfg(target_os = "windows")]
+            ModifierKeyCode::LeftSuper => write!(f, "Left Windows"),
+            #[cfg(not(any(target_os = "macos", target_os = "windows")))]
+            ModifierKeyCode::LeftSuper => write!(f, "Left Super"),
+
+            #[cfg(target_os = "macos")]
+            ModifierKeyCode::RightControl => write!(f, "Right Control"),
+            #[cfg(not(target_os = "macos"))]
+            ModifierKeyCode::RightControl => write!(f, "Right Ctrl"),
+
+            #[cfg(target_os = "macos")]
+            ModifierKeyCode::RightAlt => write!(f, "Right Option"),
+            #[cfg(not(target_os = "macos"))]
+            ModifierKeyCode::RightAlt => write!(f, "Right Alt"),
+
+            #[cfg(target_os = "macos")]
+            ModifierKeyCode::RightSuper => write!(f, "Right Command"),
+            #[cfg(target_os = "windows")]
+            ModifierKeyCode::RightSuper => write!(f, "Right Windows"),
+            #[cfg(not(any(target_os = "macos", target_os = "windows")))]
+            ModifierKeyCode::RightSuper => write!(f, "Right Super"),
+        }
+    }
+}
```

Comment:
This doesn't compile for me on linux:
```rust
922 |     fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
    |        ---                                       ^^^^^^^^^^^ expected `Result<(), Error>`, found `()`
    |        |
    |        implicitly returns `()` as its body has no tail or `return` expression
    |
    = note:   expected enum `Result<(), std::fmt::Error>`
            found unit type `()`
```

Reply:
Looks like line 922 is the problem, taking a look.

Reply:
https://github.com/crossterm-rs/crossterm/pull/888

Reply:
It's just a cfg issue, because it doesn't contain any return for non-mac.

Reply:
yep - there was an accidentally left in cfg, which I didn't notice because the checks didn't run and I was developing on a mac. I suspect it was also broken on windows.